components/Secret.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { StyleSheet, View, Text, Button } from 'react-native'
+
+import SecretsService from '../lib/secretsService'
+import { useAuthentication } from '../context/authentication'
+import { useSecrets } from '../context/secrets'
+
+const styles = StyleSheet.create({
+  container: {
+    border: '1px solid black',
```

Comment:
this is CSS, doesn't work with RN stylesheets

Reply:
Yup! this is all gone now!